marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/mappingWithInterceptorsAndCustomHook.spec.tsx

```diff
@@ -146,7 +146,7 @@ describe("Create and verify load steps, map step and flows with interceptors & c
     mappingStepDetail.setXpathExpressionInput("shipRegion", "ShipRegion");
     mappingStepDetail.setXpathExpressionInput("shippedDate", "ShippedDate");
     cy.findByTestId("shippedDate-mapexpression").blur();
-    curatePage.dataPresent().should("be.visible");
+    cy.waitUntil(() => curatePage.dataPresent().should("exist"));
```

Comment:
Same thing here for the waitUntil

Reply:
Discussed with Camila that this will be investigated in a separate task, resolving.